docs/vm-images.md

```diff
@@ -1,20 +1,60 @@
 # Supported and Tested VM Images
 
+* [General Information](#general-information)
 * [HPC CentOS 7](#hpc-centos-7)
 * [HPC Rocky Linux 8](#hpc-rocky-linux-8)
 * [Debian 11](#debian-11)
 * [Ubuntu 20.04 LTS](#ubuntu-2004-lts)
 * [Windows](#windows)
 * [Other Images](#other-images)
+* [Slurm on GCP](#slurm-on-gcp)
 
 For information on customizing VM images with extra software and configuration
 settings, see [Building Images](image-building.md).
 
 Please see the [blueprint catalog](https://cloud.google.com/hpc-toolkit/docs/setup/hpc-blueprint-catalog) for examples.
 
-For Slurm images, please see [SchedMD's GitHub repository](https://github.com/SchedMD/slurm-gcp/blob/master/docs/images.md#public-image).
+## General Information
 
-## HPC CentOS 7
+### Instance Image
```

Comment:
I'm not convinced that this new section is helping this PR especially as we have not yet harmonized the dtype of instance_image into an object of defined fields.